internals/getCsp.js

```diff
@@ -0,0 +1,29 @@
+const csp = `
+    default-src 'none';
+    script-src
+      'self'
+      'report-sample';
+    style-src
+      'self'
+      fonts.googleapis.com
+      'unsafe-inline'
+      'report-sample';
+    font-src
+      'self'
+      fonts.gstatic.com;
```

Comment:
Can you please look at https://github.com/oasisprotocol/oasis-wallet-web/issues/526
Eventually we should move away from connecting to Google services.

Reply:
+1

Added this and other future CSP improvements from https://github.com/oasisprotocol/oasis-wallet-web/pull/236 as TODO comment